ML-for-S2S/ML-for-Derecho/train-torch-unet-s2s-gust.py

```diff
@@ -0,0 +1,399 @@
+import argparse
+import numpy as np
+import pandas as pd
+import xarray as xr
+from datetime import timedelta
+import matplotlib.pyplot as plt
+import xskillscore as xs
+
+import torch
+import torch.nn.functional as F
+from torch.utils.data import DataLoader
+import torch.nn as nn
+import torch.optim as optim
+
+import torch_funcs
+import torch_s2s_dataset_gust
+
+############################################################################
+############################################################################
+
+parser = argparse.ArgumentParser(description='Training UNET for bias-correcting a field (tas2m/prsfc) from CESM.')
+
+parser.add_argument("--variable",choices=["tas2m", "prsfc"],required=True,type=str, help="Variable to be corrected.")
+parser.add_argument("--week",choices=["1", "2", "3","4","5","6"],required=True,type=str, help="Week (lead time) predicted.")
```

Comment:
let's change these numbers and type into integers -- so 
`... choices=[1, 2, 3, 4, 5, 6], required=True, type=int, ...`